### PR TITLE
mackup: 0.8.40 -> 0.8.41

### PR DESCRIPTION
**Pull request by luftmensch-luftmensch**
## Description of changes
[Release notes](https://github.com/lra/mackup/releases/tag/0.8.41)

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by smancill** [comment]
Result of `nixpkgs-review pr 333199` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>mackup</li>
    <li>mackup.dist</li>
  </ul>
</details>

**Comment by luftmensch-luftmensch** [comment]
I don't have a darwin machine to test with

**Comment by smancill** [comment]
It's just a single test that calls `pgrep`. But `procps` doesn't have a `pgrep` binary on Darwin. The only solution is to ignore the test.

**Comment by luftmensch-luftmensch** [comment]
@smancill I've added the failing test (`test_is_process_running`) to `disabledTests`

**Comment by smancill** [comment]
Result of `nixpkgs-review pr 333199` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>mackup</li>
    <li>mackup.dist</li>
  </ul>
</details>

**Review by smancill — APPROVED** [review]

File: pkgs/by-name/ma/mackup/package.nix [hunk 1]
```diff
@@ -3,29 +3,19 @@
...
-  version = "0.8.40";
+  version = "0.8.41";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "lra";
     repo = "mackup";
-    rev = "refs/tags/${version}";
-    hash = "sha256-hAIl9nGFRaROlt764IZg4ejw+b1dpnYpiYq4CB9dJqQ=";
+    rev = "${version}";
```
Comment (SuperSandro2000) [inline pkgs/by-name/ma/mackup/package.nix:15]: This should stay as refs/tags/... is guaranteed to be a tag. Just using plain version could also be a branch.

**Pull request merged**